ex02.ex

```diff
@@ -53,7 +53,14 @@ defmodule Ex02 do
   # if the first two elements of a list are equal                #
   ################################################################
 
-  first2equal = your_anonymous_function([])
+  first2equal = fn (list) ->
```

Comment:
0

This fails the second test.

I was looking for something like

~~~
def first2equal([a,a_]), do: true
def first2equal(_), do: false
~~~